README.md

```diff
@@ -92,6 +92,15 @@ Please see Hazelcast Node.js [code samples](https://github.com/hazelcast/hazelca
 
 You can also refer to Hazelcast Node.js [API Documentation](http://hazelcast.github.io/hazelcast-nodejs-client/api/current/docs/).
 
+# Configuration
+
+You can configure Hazelcast Node.js Client declaratively (JSON) or programmatically (API).
+* Programmatic configuration
+* Declarative configuration (JSON file)
+
```

Comment:
I don't think the bulleted items are needed here since they are mentioned in the introduction sentence just above them.